spec/spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 require "dotenv/load"
-require "simplecov"
-require "zonebie/rspec"
+# require "simplecov"
```

Comment:
Why are these added but commented out?

Reply:
it's a combination of me forgetting to uncomment it and wanting to remove `zonebie` from our test suite. Uncommented!